### PR TITLE
add PIS postman examples

### PR DESCRIPTION
**Pull request by KunRen1010**

### CODE REVIEW

**Comment by KunRen1010** [comment]
also fixed this
https://github.com/enablebanking/enablebanking-api-samples/issues/20

**Comment by fed239** [comment]
Couple suggestions:

Screenshot 2024-11-25 at 16 53 52

Screenshot 2024-11-25 at 16 52 38

File: postman_example/Enable Banking.postman_collection.json [hunk 1]
```diff
@@ -210,6 +211,211 @@
...
+						"com"
+					],
+					"path": [
+						"payments"
+					]
+				}
+			},
+			"response": [
+				{
+					"name": "HandelsBanken (sandbox)",
```
Comment (fed239) [inline postman_example/Enable Banking.postman_collection.json:250]: I'd name it

SE Handelsbanken DOMESTIC personal

i.e. {country} {ASPSP name} {payment type} {psu type}

we could also add auth method name at the end, but not sure if it's absolutely necessary

File: postman_example/Enable Banking.postman_collection.json [hunk 2]
```diff
@@ -210,6 +211,211 @@
...
+							]
+						}
+					},
+					"_postman_previewlanguage": "Text",
+					"header": [],
+					"cookie": [],
+					"body": "{\n    \"payment_id\": \"075d85d2-1683-4b77-9582-1a99f1708146\",\n    \"status\": \"RCVD\",\n    \"url\": \"https://tilisy-sandbox.enablebanking.com/pis/start?payment_id=075d85d2-1683-4b77-9582-1a99f1708146\"\n}"
+				},
+				{
+					"name": "SwedBank (sandbox)",
```
Comment (fed239) [inline postman_example/Enable Banking.postman_collection.json:293]: same as above

File: postman_example/Enable Banking.postman_collection.json [hunk 3]
```diff
@@ -210,6 +211,211 @@
...
+						"enablebanking",
+						"com"
+					],
+					"path": [
+						"payments",
+						"075d85d2-1683-4b77-9582-1a99f1708146"
+					]
+				}
+			},
+			"response": [
```
Comment (fed239) [inline postman_example/Enable Banking.postman_collection.json:357]: not sure if responses add a lot of value here. maybe we could keep them, but I'm afraid the may confuse more than explain, also maintaining them require additional efforts

**Pull request merged**